src/metabase/public_settings.clj

```diff
@@ -421,7 +421,7 @@
 
 (defsetting start-of-week
   (deferred-tru "This will affect things like grouping by week or filtering in GUI queries.
-  It won''t affect SQL queries.")
+  It won''t affect most SQL queries, although it is used to set the WEEK_START session variable in Snowflake.")
```

Comment:
I know this was already multiline in the first place but I'm 98% sure `xgettext` can't pick up multiline i18n forms -- although maybe it actually does work here since the line break is in the string itself? We could try to be sure tho. I did put some code in `defsetting` so we can deal with this situation -- you can wrap multiple `deferred-tr*` forms in a `str` to concatenate them in `defsetting`. Breaking this up into multiple i18n strings is a bit awkward, but better than nothing.

It doesn't seem like it would be _that_ hard for us to replace `xgettext` with something hand-rolled that could walk our code and find the i18n forms regardless of the number of lines FWIW. Maybe something to do in the future

Reply:
Fixed by copying what was done for `show-database-syncing-modal` for the multiline docstring